### PR TITLE
Filter useless buffers in Helm-mini, add command to unfiltered list buffers

### PR DESCRIPTION
**Pull request by deb0ch**
Part of the work for #11545

### CODE REVIEW

**Comment by deb0ch** [comment]
ping ?

File: layers/+completion/helm/funcs.el [hunk 1]
```diff
@@ -622,3 +622,11 @@ to buffers)."
   (interactive)
   (let (helm-candidate-number-limit)
     (helm-themes)))
+
+;; Buffers ---------------------------------------------------------------------
+
+(defun spacemcacs/helm-buffers-list-unfiltered ()
```
Comment (duianto) [inline layers/+completion/helm/funcs.el]: Function name prefix typo: `spacemcacs`
same with the function call for the `bU` key binding on line 85.

Comment (deb0ch) [reply layers/+completion/helm/funcs.el]: Done ! ✨

**Comment by deb0ch** [comment]
Ping @syl20bnr @TheBB @d12frosted @bmag @JAremko 

I just realised that I have been using this for about a year when switching branches and actually missed the functionnality a few times, so this `SPC b U` is definitively useful.

Especially for cleaning up your buffers, as `persp`, `magit` and others tend to leave a lot of garbage behind them and if, like me, you save your layouts / workspaces between each session you end up with a very slow Spacemacs.

**Comment by syl20bnr** [comment]
@deb0ch :+1: Thank you for the ping and taking the time to address parts of https://github.com/syl20bnr/spacemacs/issues/11545

Merging this in develop branch and I'll update the change log.

**Pull request merged**